src/documentation/pages/components-wip/tabs/examples/_contained-component-lumen-centurylink.vue

```diff
@@ -0,0 +1,117 @@
+<template lang="pug">
+  <ComponentExample title="Components contained" id="components-contained-lumen-centurylink" :tabs="exampleTabs">
+    p.-text(slot="example-description")
+      | As navigation component is built from other primitive Chi components, most of the components behavior
+      | can be replicated on the navigation component. For example, you can use the <code>-animate</code> class
+      | on the dropdowns to make the chevron rotate when activated.
+
+    <Wrapper slot="example">
+      ul.chi-tabs.chi-navigationExample#navigation-components-contained(ref="example__tabs_navigation_components_contained")
```

Comment:
Why would you use a `Wrapper` here?

```suggestion
      ul.chi-tabs.chi-navigationExample#navigation-components-contained(ref="example__tabs_navigation_components_contained" slot="example")
```

Check this in every example.